apps/api/src/app/subscribers/usecases/get-subscribers/get-subscriber.usecase.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { SubscriberRepository } from '@novu/dal';
+import { GetSubscribersCommand } from './get-subscriber.command';
+
+@Injectable()
+export class GetSubscribers {
+  constructor(private subscriberRepository: SubscriberRepository) {}
+
+  async execute(command: GetSubscribersCommand) {
+    return this.subscriberRepository.find({
```

Comment:
maybe add an await here?

Reply:
Another thing might be problematic is that we might need to have a pagination support here just as we do with activities or templates. In case someone will have a lot of subscriber this will return a huge payload and might cause us a lot of performance issues :)

Reply:
yeah sure I do not know if it makes any diffrence...?

Reply:
> Another thing might be problematic is that we might need to have a pagination support here just as we do with activities or templates. In case someone will have a lot of subscriber this will return a huge payload and might cause us a lot of performance issues :)

I was thinking about this and was thinking that @scopsy will say it if he think it is needed ;)